### PR TITLE
Emmeka's Portfolio: Accessibility Changes

### PR DESCRIPTION
**Pull request by roniy68**
# Highlights 🥇 

## What We have Done: 

> # With Issues
 - Image text alternatives - changes
 - Update readme file - collaborator @roniy68 @MarkoKermi 

> # No Issues: 
 - Page titles - no issues found
 - Headings - no issues found
 - Color contrast - no issues found
 - Resize - no issues found
 - Interaction - no issues with keyboard control
 - Moving, Flashing, or Blinking Content - no issues found
 - Multimedia (video, audio) alternatives- no issues found
 - Basic structure

deploy 

## Followed Rules: 
- Made sure [no linter errors](https://github.com/microverseinc/linters-config)
-  Followed [Github Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)
-  Documented [professionaly](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
-  Added Contribution names

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -103,6 +104,8 @@ <h3><a class="textdecoration font1" href="#">Tonic</a></h3>
                     <li><a class="textdecoration font3" href="#">Back End Dev</a></li>
                     <li class="textdecoration"><a href="#"><img src="./img/mid_dot.jpg" alt=""></a></li>
                     <li><a class="textdecoration font3" href="#">2015</a></li>
+                                    <!-- Image text alternativ should be added -->
+
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to the alternative text `alt` to describe your image as required.

File: index.html [hunk 2]
```diff
@@ -166,9 +170,9 @@ <h3><a class="textdecoration font1" href="#">Multi-Post Stories</a></h3>
                 <ul>
 
                     <li><a class="textdecoration font2" href="#">CANOPY</a></li>
-                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=" "></a></li>
+                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=""></a></li>
                     <li><a class="textdecoration font3" href="#">Back End Dev</a></li>
-                    <li><a class="textdecoration" href="#"><img src="./img/mid_dot.jpg" alt=" "></a></li>
+                    <li><a class="textdecoration" href="#"><img src="./img/mid_dot.jpg" alt=""></a></li>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to the alternative text `alt` to describe your image as required.

File: index.html [hunk 3]
```diff
@@ -194,9 +198,9 @@ <h3><a class="textdecoration font1" href="#">Multi-Post Stories</a></h3>
                 <ul class="ul1">
 
                     <li><a class="textdecoration font2" href="#">CANOPY</a></li>
-                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=" "></a></li>
+                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=""></a></li>
                     <li><a class="textdecoration font3" href="#">Back End Dev</a></li>
-                    <li><a class="textdecoration" href="#"><img src="./img/mid_dot.jpg" alt=" "></a></li>
+                    <li><a class="textdecoration" href="#"><img src="./img/mid_dot.jpg" alt=""></a></li>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to the alternative text `alt` to describe your image as required.

File: index.html [hunk 4]
```diff
@@ -246,9 +250,9 @@ <h3><a class="textdecoration font1" href="#">Facebook 360</a></h3>
                 <ul class="ul1">
 
                     <li><a class="textdecoration font2" href="#">CANOPY</a></li>
-                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=" "></a></li>
+                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=""></a></li>
                     <li><a class="textdecoration font3" href="#">Back End Dev</a></li>
-                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=" "></a></li>
+                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=""></a></li>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to the alternative text `alt` to describe your image as required.

File: index.html [hunk 5]
```diff
@@ -276,9 +280,9 @@ <h3><a class="textdecoration font1" href="#">Tonic</a></h3>
                 <ul>
 
                     <li><a class="textdecoration font2" href="#">CANOPY</a></li>
-                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=" "></a></li>
+                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=""></a></li>
                     <li><a class="textdecoration font3" href="#">Back End Dev</a></li>
-                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=" "></a></li>
+                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=""></a></li>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to the alternative text `alt` to describe your image as required.

File: index.html [hunk 6]
```diff
@@ -342,9 +346,9 @@ <h3><a class="textdecoration font1" href="#">Uber Navigation</a></h3>
                 <ul class="ul1">
 
                     <li><a class="textdecoration font2" href="#">CANOPY</a></li>
-                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=" "></a></li>
+                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=""></a></li>
                     <li><a class="textdecoration font3" href="#">Back End Dev</a></li>
-                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=" "></a></li>
+                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=""></a></li>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to the alternative text `alt` to describe your image as required.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @EmekaEmmanuel   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Good PR Description 👍 

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by roniy68** [comment]
# Thank you @frank1738 @levy002 

thanks

**Pull request merged**